.circleci/config.yml

```diff
@@ -250,21 +250,10 @@ workflows:
   version: 2
   "Run Tests & Checks": # Name of the workflow, which ends up displayed on GitHub's PR Check
     jobs:
-      - tests_macos:
-          name: 'Execute tests on macOS (Xcode 12.5.1, Ruby 2.6)'
-          xcode_version: '12.5.1'
-          ruby_version: '2.6.6'
-          <<: *important-branches
-      - tests_macos:
-          name: 'Execute tests on macOS (Xcode 12.5.1, Ruby 2.7)'
-          xcode_version: '12.5.1'
-          ruby_version: '2.7.3'
-          ruby_opt: -W:deprecated
-          <<: *important-branches
       - tests_macos:
           name: 'Execute tests on macOS (Xcode 13.4.1, Ruby 3.0)'
           xcode_version: '13.4.1'
-          ruby_version: '3.0.4'
+          ruby_version: '3.1'
```

Comment:
Ruby version is bumped to 3.1 here but it's inconsistent with the name `'Execute tests on macOS (Xcode 13.4.1, Ruby 3.0)'`. `3.1` is also duplicated below too. Maybe we could just remove 3.0 support from CI...?

Reply:
Oops, your point is right, it seems to be duplicated. I'll get rid of this soon. Thank you!

Reply:
Thank you for pointing this. I fixed! https://github.com/fastlane/fastlane/pull/22156